v03_pipeline/lib/reference_datasets/config.py

```diff
@@ -0,0 +1,34 @@
+from v03_pipeline.lib.model import ReferenceGenome
+from v03_pipeline.lib.reference_data.cadd import load_cadd_ht_from_raw_dataset
+from v03_pipeline.lib.reference_data_OLD.hgmd import download_and_import_hgmd_vcf
+
+CONFIG = {
```

Comment:
I was envisioning this as part of the enum class, so it would share types and constants more clearly, but I could be convinced that it is better to keep it in a different file
```
class ReferenceDataset(str, Enum):
    cadd = 'cadd'
    hgmd = 'hgmd'

    LOAD_DATASET_FUNC = 'load_parsed_dataset_func'
    VERSION = 'version'
    CONFIG = {
        cadd: {
            LOAD_DATASET_FUNC: load_cadd_ht_from_raw_dataset,
            ReferenceGenome.GRCh37: {
                VERSION: '0.0',
            },
        },
    }
```

Reply:
oh yeah I like it in the same file